docs/src/basics.md

```diff
@@ -0,0 +1,4 @@
+The AstroImages package provides a wrapper, `AstroImage`, that can wrap any AbstractArray.
+An `AstroImage` should behave like a plain array, but gains a few extra abilities.
+
+First, AstroImages are automatically displayed when returned as results in many environements, including VSCode, Jupyter, Pluto, ImageShow, and ImageInTerminal
```

Comment:
```suggestion
First, AstroImages are automatically displayed when returned as results in many environements, including VSCode, Jupyter, Pluto, ImageShow, and ImageInTerminal.
```
"First" suggests there should be a second?   But maybe this is related to the fact docs are still incomplete, as you said?